addons/mail/static/src/new/messaging.js

```diff
@@ -134,6 +136,16 @@ export class Messaging {
         });
     }
 
+    updateImStatusRegistration(partner) {
+        if (partner.im_status !== "im_partner" && !partner.is_public) {
+            this.registeredImStatusPartners.push(partner.id);
+        }
+        this.imStatusService.registerToImStatus(
+            "res.partner",
+            markRaw(this.registeredImStatusPartners)
```

Comment:
This should be `toRaw()`, as it's passed to external API (it's stored as a reactive, and should be kept as so)